Monika After Story/game/script-topics.rpy

```diff
@@ -5530,6 +5530,41 @@ label monika_rock:
             m 1ekc "Oh...that's okay, everyone has their own taste in music."
             m 1hua "Though, if you ever do decide to listen to some rock 'n' roll, I'll happily listen right alongside you."
     return "derandom"
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_standup",category=['literature'],prompt="Stand-up Comedy",random=True))
+
+label monika_standup:
+    m 2hua "You know what's a neat form of literature?"
+    m 2hub "Stand-up comedy!"
+    m 2hksdrb "...I've been saying lots of random things are literature haven't I?"
+    m 2hub "I'm starting to feel like Natsuki, or some fanatic post-modernist, ahaha!"
+    m 2esa "But seriously, there's a real craft when it comes to writing bits for stand-up."
+    m 4esd "It differs from making simple one-liner jokes, because it really needs to tell a story."
```

Comment:
```suggestion
    m 4esc "It differs from making simple one-liner jokes, because it really needs to tell a story."
    m 4eud "But at the same time, you have to make sure you don't lose your audience."
    m 4euc "So it's important to develop your ideas as much as you can, maybe even segueing into something that relates to your topic..."
    m 4eub "All the while, always keeping the audience captivated until you reach the punch line, hopefully resulting in lots of laughs."    
```